flight-delay/app/flightDelay.js

```diff
@@ -13,38 +13,35 @@ const errorFilePath       = `${root}/error.iexec`;
  *                                 ARGUMENTS                                 *
  *****************************************************************************/
 
-var [id , timestamp] = process.argv.slice(2).map(s => s);
+var [id, timestamp] = process.argv.slice(2).map(s => s);
```

Comment:
the map is not really needed here :P

Reply:
Timestamp is not processed and just passed in the result. There is absolutelly no link between this timestamp and the results provided.

A better approach would be to not include timestamp in the arguments, and put a timestamp comming provided by the api (latest update for this flight ?)